src/utilcode/ccomprc.cpp

```diff
@@ -10,7 +10,10 @@
 #include "ndpversion.h"
 
 #include "../dlls/mscorrc/resource.h"
-#include "../dlls/mscorrc/resourcestring.h"
+#ifdef PLATFORM_UNIX
+#include "resourcestring.h"
+DECLARE_STRING_RESOURCE_TABLE(STRING_RESOURCE_NAME);
```

Comment:
What is the benefit of defining the STRING_RESOURCE_NAME in the buildfile when the only usage is in this file and this file is always built with the same value of the macro?